os/fs/procfs/fs_procfslogsave.c

```diff
@@ -122,7 +122,8 @@ static int logsave_open(FAR struct file *filep, FAR const char *relpath, int ofl
 	FAR struct logsave_file_s *attr = (struct logsave_file_s *)kmm_zalloc(sizeof(struct logsave_file_s));
 	fvdbg("Open '%s'\n", relpath);
```

Comment:
I think it is better to move this log after checking the `log_dump_read_init()`, I mean after successfully open it.
Because, even fail to init, so we return `ERROR`, but the log looks all is okay.
It can make confusing to debug.